.devcontainer/devcontainer.json

```diff
@@ -11,7 +11,8 @@
         "ms-python.python",
         "ms-azuretools.vscode-docker",
         "ms-toolsai.jupyter",
-        "yahyabatulu.vscode-markdown-alert"
+        "yahyabatulu.vscode-markdown-alert",
+        "tomoki1207.pdf"
```

Comment:
What's this PDF file?

Reply:
This is the name of the VS Code extension, it's always [author's username].[individual extension name]. Please see here, that's the extension: https://marketplace.visualstudio.com/items?itemName=tomoki1207.pdf

Reply:
Oh I see, thanks!